apis/Google.Cloud.Iam.V1/Google.Cloud.Iam.V1/PolicyPartial.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Copyright 2017 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Api.Gax;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Google.Cloud.Iam.V1
+{
+    // Convenience methods for Policy
+    public partial class Policy
+    {
+        /// <summary>
+        /// Adds the specified member to the specified role. If the role does
+        /// not already exist, it is created.
+        /// </summary>
+        /// <param name="role">The role to add the member to. Must not be null.</param>
+        /// <param name="member">The member to add to the role. Must not be null.</param>
+        /// <returns><c>true</c> if the policy was changed; <c>false</c> if the member already existed in the role.</returns>
+        public bool AddRoleMember(string role, string member)
+        {
+            GaxPreconditions.CheckNotNull(role, nameof(role));
```

Comment:
Should these also check the strings are not empty?

Reply:
Yes, probably a good plan. Will change.

Reply:
Done. Will merge on green.